src/components/SingleArticle.js

```diff
@@ -0,0 +1,70 @@
+import React, { useState, useEffect } from "react";
+import axios from "axios";
+import { View, Image, Text, StyleSheet } from "react-native";
+import { EBGaramond_400Regular } from "@expo-google-fonts/eb-garamond";
+
+const SingleArticle = ({ route }) => {
+  const { articleId } = route.params;
+  console.log(articleId);
+  const [article, setArticle] = useState({});
+
+  const chooseArticle = async () => {
+    let response = await axios.get(`/articles/${articleId}`);
+    setArticle(response.data.article);
+  };
+
+  useEffect(() => {
+    chooseArticle();
+  }, []);
+  const specificArticle = article && (
+    <View>
+      <View key={article.id} className="article-title">
+        <Image
+          className={`article-image-${article.id}`}
+          source={{ uri: article.image }}
+          style={styles.image}
+        />
+        <Text id={"article-" + article.id + "-title"} style={styles.title}>
+          {article.title}
+        </Text>
```

Comment:
We could be able to re-use the ArticleCard here

Reply:
well the article card doesnt use the body of the article and it dont have to be touchable....